src/raftstore/store/peer.rs

```diff
@@ -1889,7 +1905,7 @@ impl Peer {
 
     fn read_local<T, C>(&mut self, ctx: &mut PollContext<T, C>, req: RaftCmdRequest, cb: Callback) {
         ctx.raft_metrics.propose.local_read += 1;
-        cb.invoke_read(self.handle_read(ctx, req, false, None))
+        cb.invoke_read(self.handle_read(ctx, req, false, Some(self.get_store().committed_index())))
```

Comment:
Why `read_index` should be `committed_index` instead of `None` ?  Could this function `read_local` be called when `RaftStore` deal with a message which type is `ReadIndex`?

Reply:
Seems for `read_local` the `cmd_type` will never be `CmdType::ReadIndex`. So I also think it's not necessary.

Reply:
Currently when a leader receives a `CmdType::ReadIndex` and it has a valid lease, the `RequestPolicy` will be `ReadLocal` if the `read_quorum` field of the request is false. Then a `None` value `read_index` will cause TiKV panic.

Reply:
Here assumes that we can return committed index safely when the leader gets a lease, without sending any heartbeat